package.json

```diff
@@ -28,5 +28,10 @@
     "lerna": "^5.5.4",
     "lint-staged": "^13.0.3",
     "turbo": "^1.5.5"
+  },
+  "lint-staged": {
+    "*.{tsx,jsx,ts,js,md,css,yml,json}": [
+      "prettier --write"
```

Comment:
Why prettier and not eslint? We have rules in eslint like calling out the debugher command and removing `.only` when left in. If we want to decouple them, we can remove the wrapper and run prettier and eslint separately.

Reply:
Goal is to do both. Having trouble getting eslint to work across the project but working on that next.

Reply:
Right now neither is being run project wide. Prettier just works out of the box so figured it made sense to quickly get that in place while working through eslint.